examples/tests/unit_consistency_misc.nbt

```diff
@@ -62,6 +62,8 @@ assert_eq(8e-15, 8 ppq, 1e-16)
 
 assert_eq(1 calorie, 4.184 J)
 assert_eq(1 BTU, 1055.05585262 J)
+assert_eq(1 therm, 1.05505585262e8 J)
+assert_eq(1 thermie, 4.184e6 J)
```

Comment:
That is actually different from the value quoted on Wikipedia:

> The thermie is equivalent to 1000 [kcal](https://en.wikipedia.org/wiki/Calorie), 4.1868 [MJ](https://en.wikipedia.org/wiki/Megajoule) or 3968.3 [BTU](https://en.wikipedia.org/wiki/British_thermal_unit).

 I think Wikipedia might be wrong here since their values are contradictory. According to their own linked article, 1 kcal = 4.184 J, so if 1 thermie = 1000 kcal, then 1 thermie = 4.184 MJ.

The 4.1868 value refers to some older definition of the calorie (International Steam Table calorie, 1929 and 1956).

And the only source that they quote on the Thermie article is https://www.cnrtl.fr/definition/thermie, which also has a value of 4.184 MJ.

I'm going to change the Wikipedia article, I guess.

Reply:
I opened a discussion on the Talk:Thermie page: https://en.wikipedia.org/wiki/Talk:Thermie#Should_the_value_of_the_thermie_be_4.184_MJ?

Reply:
This publication might also give some insight: [Publication:Calorie](https://www.degruyter.com/document/doi/10.1351/pac196408020095/html) (Specifically, pages 106 and 107)